.gitignore

```diff
@@ -0,0 +1 @@
+.hypothesis
```

Comment:
Remove this so there is a hypothesis folder with examples that have been tested.

Reply:
done in https://github.com/mbattifarano/inverse-astar-search/pull/1/commits/331f8ba60fff16e2b92b1d58dba4bb298b73753b